src/less/success-message/base/success-message.less

```diff
@@ -0,0 +1,61 @@
+.success-message__button,
+.success-message__fake-button {
+    .btn-base-no-font-size();
+
+    background-color: @success-message-button-color;
+    border-color: @success-message-button-border-color;
+    font-size: @font-size-16;
+    margin-top: 28px;
+    min-height: 48px;
+    padding: 15px 16px 13px;
+
+    &:hover {
+        background-color: @success-message-button-color-hover;
+        border-color: @success-message-button-color-hover;
+        color: @success-message-button-border-color;
+        text-decoration: none;
+    }
+}
+
+.success-message--full {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    position: relative;
+
+    .success-message__button,
+    .success-message__fake-button {
+        bottom: 0;
+        left: 0;
+        position: absolute;
+        width: 100%;
+    }
+}
+
+.success-message {
+    background-color: @success-message-background-color;
+    box-sizing: border-box;
+    padding: 0 9%;
+    text-align: center;
+    width: 100%;
+}
+
+.success-message__status {
```

Comment:
Do we also support the `svg` tag case?

Reply:
I was following what I saw in the notice demo.  But I didn't realize we supported svg there too, it just wasn't in the demo.  Arg... OK.  Working on it.